algebraic-types/README.md

```diff
@@ -0,0 +1,7 @@
+
+Extra tools and samples for the Bril algebraic types extension.
+
+ - `briltc`: Bril type-checker with support for algebraic types
+ - `rs2bril`: compiler from Rust syntax to Bril
+ - `samples`: sample Bril programs using algebraic types
```

Comment:
Seems like maybe these should be included in the test suite? Or are they different?